### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by mariia-kiliushina**

### CODE REVIEW

File: src/Components/Carousel/SlideOnboarding.tsx [hunk 1]
```diff
@@ -8,24 +8,26 @@ export type TSlideOnboarding = {
...
 };
 
 export const SlideOnboarding = ({item}: TSlideOnboardingProps) => {
+  const {title, content, Icon} = item;
+
   return (
     <View style={styles.container}>
       <View style={styles.imageContainer}>
-        {React.createElement(item.icon)}
+        <Icon />
```
Comment (MaximZybo) [inline src/Components/Carousel/SlideOnboarding.tsx:24]: Please leave like it was, I want it to be the same as in src/Components/Inputs/BaseInput.tsx

File: src/Components/Carousel/SlideOnboarding.tsx [hunk 2]
```diff
@@ -8,24 +8,26 @@ export type TSlideOnboarding = {
...
-  icon: React.FC<SvgProps>;
+  Icon: React.FC<SvgProps>;
 };
 
 type TSlideOnboardingProps = {
   item: TSlideOnboarding;
 };
 
 export const SlideOnboarding = ({item}: TSlideOnboardingProps) => {
+  const {title, content, Icon} = item;
```
Comment (MaximZybo) [inline src/Components/Carousel/SlideOnboarding.tsx:19]: Can change to 
export const SlideOnboarding = ({
  item: {content, icon, id, title},
}: TSlideOnboardingProps)

File: src/Navigation/types.tsx [hunk 3]
```diff
@@ -53,7 +53,10 @@ export type HomeStackParamList = {
 export type HomeStackScreenProps<T extends keyof HomeStackParamList> =
   CompositeScreenProps<
     NativeStackScreenProps<HomeStackParamList, T>,
-    RootStackScreenProps<keyof RootStackParamList>
+    CompositeScreenProps<
+      BottomTabScreenProps<AfterAuthParamList, 'HomeStack'>,
```
Comment (MaximZybo) [inline src/Navigation/types.tsx:57]: Don't add BottomTabScreens. For now there is no reason for this

Comment (mariia-kiliushina) [reply src/Navigation/types.tsx:57]: When I add my components I need navigation e.g. from Dashboard of HomeStack to Bills of BillsStack without it I'am having TS error

**Pull request merged**